exercises/concept/lucians-luscious-lasagna/.meta/design.md

```diff
@@ -0,0 +1,36 @@
+# Design
+
+## Learning objectives
+
+-   Know how to define a function
+-   Know how to return a value from a function
+
+## Out of scope
+
+-   TODO
```

Comment:
We can probably just delete the TODO sections of this document, which as I understand it is meant to guide the implementation of an exercise, but isn't used for much after the exercise is implemented.